src/styles/blocks/icon.scss

```
@@ -0,0 +1,55 @@
+.icon {
+  display: block;
+  width: 24px;
+  height: 24px;
+
+  @include hover(transform, scale(1.2));
+
+  @include onTablet {
+    height: 32px;
+    width: 32px;
+  }
+
+  &--phone {
+    background-image: url(../images/icons/Icon-Phone-call.svg);
+    background-size: cover;
+    position: relative;
+
+    &:hover {
+      &::before {
+        content: "+1 212-535-7710; +1 212-923-3700";
```

You can make it like that, and provide the values from mockup

```suggestion
      &::before {
      // eslint-disable-next-line
        content: "+1 212-535-7710; +1 212-923-3700";
```

Delete the comment in code after to will fix it.